### DISCUSSION TITLE
How to get Started Contributing?

### DISCUSSION THREAD

**Discussion by cookacounty** [rggen/rggen#71] [Q&A]
Could you provide some basic information on how to get started contributing? I've cloned the area and did a bundle install but I'm not sure where to go next from there. Some insight into your work area would help! 
* How would I setup a work area to work on my own custom plugin
* How would I setup a work area to work on the interpreter (like excel)?
* What IDE do you use? How do you set break points to debug code?

Any insight you could provide would go a long ways!

**Comment by taichi-ishitani** [comment]
> How would I setup a work area to work on my own custom plugin
> How would I setup a work area to work on the interpreter (like excel)?

I added wikis describing how to create your own plugin project.
(now under construction)
https://github.com/rggen/rggen/wiki/Setup-Plugin-Project
https://github.com/rggen/rggen/wiki/Add-New-Bitfield-Type
How about these wikis?

> What IDE do you use? How do you set break points to debug code?

I'm using VS code with the [ruby plugin](https://marketplace.visualstudio.com/items?itemName=rebornix.Ruby).
`ruby-debug-ide` and `debase` gems also needed for debugging on VS code.

**Reply by taichi-ishitani** [reply]
I also put contribution guide.
https://github.com/rggen/rggen/blob/master/CONTRIBUTING.md
